### PR TITLE
feat: Add Person, Student and Teacher classes

### PR DESCRIPTION
**Pull request by gvgesanto2**
## Description

**Task:** https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md

This PR makes the following changes:
- Sets up the GitHub Actions and the [Rubocop](https://github.com/rubocop/rubocop) linter.
- Creates the ``Person``, ``Student`` and ``Teacher`` classes.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [X] My code follows the style guidelines of this project (no linters errors)
- [X] I have checked my code and corrected any misspellings
- [X] I have deleted any unused code as comments in my codebase

### CODE REVIEW

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @gvgesanto2 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- good pr
- passing linters 👍🏾 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age:, name: 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```
Comment (alexander16108) [inline person.rb]: you should have an id in your person class file 

`@id =Random.rand(1..1000)
`

**Comment by gvgesanto2** [comment]
## Description

**Task:** https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md

This PR addresses the changes requested by the [last review](https://github.com/gvgesanto2/microverse-school-library/pull/1#discussion_r958973720).

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @gvgesanto2 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done making the changes requested by the previous reviewer. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**